bin/.gitignore

```diff
@@ -1 +0,0 @@
-/com/
```

Comment:
I don't know what any of this means. Those files just showed up when I moved my code to this repository.